src/WebJobs.Script.WebHost/Diagnostics/LinuxContainerEventGenerator.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    internal class LinuxContainerEventGenerator : IEventGenerator
```

Comment:
Can we just call this `ConsoleEventGenerator` or something along those lines?

Reply:
There's another set of changes coming soon where not all of these events will go to console. E.g. the function executed event (currently not implemented) will go to another location. I can explain all of this more in person :)